homeassistant/components/remootio/strings.json

```diff
@@ -0,0 +1,42 @@
+{
+  "config": {
+    "flow_title": "Remootio",
+    "step": {
+      "user": {
+        "title": "Remootio integration :: Settings",
+        "description": "To use this integration with your Remootio device it has to be connected to your Wi-Fi, must have a fixed IP address assigned and a status sensor installed. Furthermore the API access must be enabled on the device.\n\nTo enable the API access on the device you need to access the device using the master key via the Remootio app, afterwards go to the _Websocket API_ settings in the _Device software_ section of the device settings and enbale the API with logging.\nAfter enabling the API the app will display the device's IP address and the credentials needed for API access.\n\nPlease enable the API on your Remootio device and enter the IP address and credentials shown by the app. Instead of the IP address you can also enter the host name of your Remootio device if you have configured one on your router or DNS server for it.",
+        "data": {
+          "host": "Host (IP address or host name)",
+          "secret__api_secret_key": "API Secret Key",
+          "secret__api_auth_key": "API Auth Key",
+          "device_class": "Controlled device's class"
+        }
```

Comment:
Please look into adding a `data_description` dictionary where you can give some more details, while also shortening the description above.